compiler/src/iree/compiler/Codegen/LLVMGPU/Verifiers.cpp

```diff
@@ -200,8 +205,11 @@ LogicalResult verifyGPUMatmulPipeline(
 
   // Instruction shape in number of elements in M, N, and K dim.
   SmallVector<int64_t> instructionShape;
-  getInstructionShape(pipeline, lhsType.cast<ShapedType>().getElementType(),
-                      instructionShape);
+  if (getInstructionShape(op, pipeline,
+                          lhsType.cast<ShapedType>().getElementType(),
+                          instructionShape)
```

Comment:
that should be `if (failed(getInstructionShape(...`?

Reply:
never mind

Reply:
Resolved! Thanks